db/migrate/20190724042815_add_deleted_at_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeletedAtToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :deleted_at, :timestamp
```

Comment:
Can we make this a `datetime`, not a `timestamp` pls? I prefer to store dates as dates.

Reply:
`timestamp` and `datetime` in Rails is the same thing.